src/multi_player.c

```diff
@@ -173,6 +185,12 @@ bool multi_player(SDL_Window* p_window,option_items_t difficulty)
 						}
 
 					}
+					else if (is_bullet_hit(&map, PLAYER_1 == true)) {
+						take_heart(&map.player[PLAYER_1]);
```

Comment:
After take_heart you should check the player's heart to see if its zero. If it is zero the game should be over.
Like what we did after is_player_hit.